datasets/README.md

```diff
@@ -108,3 +105,66 @@ python3 spd_datasets.py \
     --hub_repository=diarizers-community/callhome \
 ```
 
+
+## 2. Generate a synthetic dataset compatible with diarizers: 
+
+### Installation
+
+To use the synthetic dataset pipeline, first install `diarizers`: 
+
+```sh
+git clone https://github.com/huggingface/diarizers.git
+cd diarizers
+pip install -e .
+```
+
+To augment your synthetic datas with noise, you need to use background noise and room impulse response datasets. Here are suggested datasets and how to download them: 
+
+- Background Noise dataset: [WHAM!](http://wham.whisper.ai/). To download: 
+
+```
+wget https://my-bucket-a8b4b49c25c811ee9a7e8bba05fa24c7.s3.amazonaws.com/wham_noise.zip
+unzip wham_noise.zip
+```
+
+- Room Impulse Response dataset: [MIT-ir-survey](https://mcdermottlab.mit.edu/Reverb/IR_Survey.html). To download: 
+
+```
+wget https://mcdermottlab.mit.edu/Reverb/IRMAudio/Audio.zip
+unzip Audio.zip
+```
+
+### How to use? 
+
+To generate synthetic datasets, you will need to specify a few parameters. The different parameters are explained in the [synthetic_dataset.py](synthetic_dataset.py) script. You can generate a synthetic speaker diarization dataset with this example: 
+
+```bash
+python3 synthetic_dataset.py \
+    --dataset_name="mozilla-foundation/common_voice_17_0" \
+    --subset="validated" \
+    --split="ja" \
+    --speaker_column_name="client_id" \
+    --audio_column_name="audio" \
+    --min_samples_per_speaker=10 \
+    --nb_speakers_from_dataset=-1 \
+    --sample_rate=16000 \
+    --nb_speakers_per_meeting=3 \
+    --num_meetings=200 \
+    --segments_per_meeting=16 \
+    --normalize=True \
+    --augment=False \
+    --overlap_proba=0.3 \
+    --overlap_length=3 \
+    --random_gain=False \
+    --add_silence=False \
+    --silence_duration=3 \
+    --silence_proba=3 \
+    --denoise=False \
+    --bn_path=None \
+    --ir_path=None \
+    --num_proc=2 \
+    --push_to_hub=True \
+    --hub_repository='test'
+```
+
+Find more informations on how to use 🤗 Diarizers synthetic speaker diarization pipeline in this notebook: [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1kaKv5Qa2dUuEwyLoFeh5mCwgy8O_ZdYA#scrollTo=27RrvTZte4BF). 
```

Comment:
Unfortunately I don't have access to the notebook!

Reply:
Just shared!